### DISCUSSION TITLE
Hilfe bei Install benötigt aktuellste

### DISCUSSION THREAD

**Discussion by derchan** [MiczFlor/RPi-Jukebox-RFID#1338] [Q&A] [answered]
Hallo, 
ich bin jetzt seit ein paar Wochen hier aktiv. Bisher war die frustration ganz in Ordnung weil ich so einen Grund hatte mich langsam in den Code einzuarbeiten. Ich sitze aber jetzt seit Tagen an einem Problem. 
Auf dem develop und master Branch. 
Im aktuellen neuen Install bin ich aber auf dem development Branch. 

Setup:
Ich verwende einen RPi ZeroW(jetzt grade in der Box) und einen Rpi 3A+ 

- Lautsprecher Trust Leto
- USB 2.0 Soundkarte mit Stereo von goobay. Wurde aber auch über klinke und eine andere USB Soundkarte getestet 
- Pimoroni OnOff SHIM
- USB HUB
- LCD 1602 + I2C HD44780 
- Neuftech USB RFID
- EG STARTS Zero Delay USB-Encoder
- 2021-01-11-raspios-buster-armhf-lite
- Spotify edition 
- develop stand 13.03.2021

Erstmal die beiden Schlimmsten Probleme die leider auch erst seit den Neuinstall auftreten. 
Wenn ich mit Spotify installiere kann ich meine Playlisten unter der modipy Oberfläche(port 6680) abspielen. Das geht leider nicht über die Normale Oberfläche auf (port 80) d.h. Ich kann zwar wenn es einmal über modipy gestartet ist ein Lied stoppen und fortführen. Ich kann aber nicht die Playlisten direkt Starten über diese Oberfläche oder dadurch natürlich auch nicht starten über RFID zu starten.

Dann Habe ich noch ein Problem das mopidy irgendwie die Lokale Dateien nicht mehr ein ließt auch seit 1-2 Wochen. Nachdem es kurzweilig nach manuellen scan in #1316 gelöst wurde. 

Ist das Problem nach Neuinstallation neu aufgetreten.
Wenn ich `mopidy local scan` oder `sudo mopidyctl local scan` ausführe dann bekomme ich folgende Fehlermeldung:
`mopidy läuft unter WARNING  [MainThread] mopidy_local.commands Failed scanning file:///home/pi/RPi-Jukebox-RFID/shared/audiofolders/papa/Baba_Yetu_feat_Malukah.mp3: Timeout after 1000ms
WARNING  [MainThread] mopidy_local.commands Failed scanning file:///home/pi/RPi-Jukebox-RFID/shared/audiofolders/placeholder: gst-stream-error-quark: Stream contains no data. (4)`

Was ich bisher versucht habe. 
- Update von mopidy auf 3.56.1
- Rechte kontrolliert (das Zielverzeichnis ist user pi:www-data oder www-data:www-data)
- mopidy user zu den gruppen www-data und pi hinzugefügt. 

Kann mir hier jemand noch Tipps zum Debuggen geben? 
Hat jemand eventuell eine Idee woran es liegen könnte?
Kann jemand mit einer anderen SD-karte einen Neuinstall machen um zu verifizieren das es nur an mir liegt?

**Comment by s-martin** [comment, marked as answer]
Was sagt das logfile?
`journalctl -u mopidy `

**Reply by derchan** [reply]
Danke nach probieren, log lesen und wieder probieren habe ich es gefunden.

Muss man das erstmal wissen. 
Auschlaggebend war aus dem Log:
`Mar 18 20:32:20 chanorama mopidy[1029]: INFO     [SpotifyBackend-6] mopidy_spotify.lookup Failed to lookup 'spotify:playlist:xxx': Playlist Web API lookup failed`

Das ganze lag anscheinend daran das ich aus dem APP ACCESS in Spotify die "Mopidy extensions"  entfernt haben muss. Vermutlich da ich dachte, ich brauch nur die "Iris Mopidy" Rechte in Spotify. 
Es müssen also in den App Berechtigungen diese beiden Rechte vorhanden sein. 
Dann funktioniert es auch!
